scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl

```diff
@@ -801,6 +811,90 @@ task SubpopulationAlleleNumber {
     }
 }
 
+task DuplicateAnnotations {
+    input {
+        String query_project_id
+        String fq_vat_table
+        String last_modified_timestamp
+    }
+    # there are vids with duplicate annotations
+    # make sure there aren't matching sites with mis-matched ANs and matching variants with mis-matched ACs
+
+    String pf_file = "pf.txt"
+    String results_file = "results.txt"
+
+    command <<<
+        set -e
+
+        echo "project_id = ~{query_project_id}" > ~/.bigqueryrc
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv 'SELECT * from
+        (SELECT contig, position, gvs_all_an, COUNT(DISTINCT gvs_all_an) AS an_count FROM `~{fq_vat_table}`
+        group by contig, position, gvs_all_an)
+        where  an_count >1
+        ' > bq_an_output.csv
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv 'SELECT * from
+        (SELECT contig, position, vid, gvs_all_ac, COUNT(DISTINCT gvs_all_ac) AS ac_count FROM `~{fq_vat_table}`
+        group by contig, position, vid, gvs_all_ac)
+        where  ac_count >1
+        ' > bq_ac_output.csv
+
+        # get number of lines in bq query output
+        NUMANRESULTS=$(awk 'END{print NR}' bq_an_output.csv)
+        NUMACRESULTS=$(awk 'END{print NR}' bq_ac_output.csv)
+
+
+        echo "false" > ~{pf_file}
+        # if the result of the query has any rows, that means there are sites with mis-matched gvs_all_an
+        if [[ $NUMANRESULTS = "0" ]]; then
+          echo "The VAT table ~{fq_vat_table} has a correct calculation for AN and the AN of subpopulations" > ~{results_file}
+          echo "true" > ~{pf_file}
+       # if the result of the query has any rows, that means there are variants with mis-matched gvs_all_ac
+        else if [[ $NUMACRESULTS = "0" ]]; then
```

Comment:
I believe this is causing the parse error George describes. Alternatively this could stay as an `else if` with a matching `fi` added, but I think the change I'm proposing is closer to the original intent.
```suggestion
        elif [[ $NUMACRESULTS = "0" ]]; then
```